app/assets/stylesheets/form.scss

```diff
@@ -11,13 +13,51 @@ form {
     display: inline-block;
     width: auto;
     margin-right: 5px;
+    border-radius: 0px;
   }
 }
 
+#sleeplog_in_bed_time {
+  width: 140px;
+}
+
+#sleeplog_fall_asleep_time {
+  @extend #sleeplog_in_bed_time;
+}
+
+#sleeplog_wake_up_time {
+  @extend #sleeplog_in_bed_time;
+}
+
+#sleeplog_date {
+  @extend #sleeplog_in_bed_time;
+}
+
+#sleeplog_hours {
+  @extend #sleeplog_in_bed_time;
+}
+
+#sleeplog_quality {
+  @extend #sleeplog_in_bed_time;
+}
+
+#user_reminder_time {
```

Comment:
Avoid using id selectors<br>Selector `user_reminder_time` should be written in lowercase with hyphens